Automattic/wp-calypso
# [Private media files] Support woocommerce extension

#### Changes proposed in this Pull Request

This PR is related to [private atomic sites project](https://wp.me/paObgF-Ui). It add support for remote private images to Woocommerce extension.

<img width="976" alt="Zrzut ekranu 2020-03-9 o 14 33 31" src="https://user-images.githubusercontent.com/205419/76217589-f9512700-6212-11ea-82b4-1787ef5a130f.png">

#### Testing instructions

* Test on calypso.localhost
* Apply D39236-code (proxy provider script)
* Sandbox the REST API
* Create a new store site
* Go to Calypso > Store
* Make the site public
* Create a new product, upload an image, save it
* Make the site private in another tab
* Go to products list, inspect thumbnails, confirm the `src` is set to `blob: <identifier>`
* Edit the product, inspect images, confirm the `src` is set to `blob: <identifier>`
* Confirm the images are being displayed properly
